src/main/java/com/bandwidth/iris/sdk/IrisClient.java

```diff
@@ -1,186 +0,0 @@
-package com.bandwidth.iris.sdk;
-
-import com.bandwidth.iris.sdk.model.BaseModel;
-import com.bandwidth.iris.sdk.model.BaseResponse;
-import com.bandwidth.iris.sdk.utils.XmlUtils;
-import org.apache.commons.lang3.StringUtils;
-import org.apache.http.Header;
-import org.apache.http.HttpResponse;
-import org.apache.http.auth.AuthScope;
-import org.apache.http.auth.Credentials;
-import org.apache.http.auth.UsernamePasswordCredentials;
-import org.apache.http.client.methods.*;
-import org.apache.http.client.utils.URIBuilder;
-import org.apache.http.entity.ContentType;
-import org.apache.http.entity.FileEntity;
-import org.apache.http.entity.StringEntity;
-import org.apache.http.impl.client.DefaultHttpClient;
-import org.apache.http.util.EntityUtils;
-
-import javax.xml.stream.XMLInputFactory;
-import java.io.File;
-import java.net.URISyntaxException;
-import java.util.HashMap;
-import java.util.Map;
-
-/**
- * Created by sbarstow on 9/22/14.
- */
-public class IrisClient {
-
-    private static String defaultUri = "https://api.inetwork.com/v1.0";
-    private static String defaultVersion = "v1.0";
-    protected DefaultHttpClient httpClient;
```

Comment:
with the latest version (4.5.x) of apache HTTP client `DefaultHttpClient` became deprecated. Consider upping version and use of `CloseableHttpClient` (or `org.apache.http.client.HttpClient`) instead